config.js

```diff
@@ -264,6 +264,11 @@ var config = {
     // a call is hangup.
     // enableClosePage: false,
 
+    // Enabling pre join page will add an additional step before starting the meeting,
+    // where the user can configure its devices and choose the way he
+    // joins audio (by phone/or web).
+    // prejoinPageEnabled: false,
```

Comment:
Can you please remove this until we have the actual pre join page? It's not on the whitelist either, but we can add it later when the full pre-join experience lands.